apache-jstl/src/test/java/org/eclipse/jetty/jstl/JspConfig.java

```diff
@@ -33,7 +33,7 @@ public static void init(WebAppContext context, URI baseUri, File scratchDir)
     {
         context.setAttribute("javax.servlet.context.tempdir", scratchDir);
         context.setAttribute("org.eclipse.jetty.server.webapp.ContainerIncludeJarPattern",
- ".*/javax.servlet-[^/]*\\.jar$|.*/servlet-api-[^/]*\\.jar$|.*javax.servlet.jsp.jstl-[^/]*\\.jar|.*taglibs-standard-impl-.*\\.jar");
+ ".*/javax.servlet-[^/]*\\.jar$|.*/[^/]*servlet-api-[^/]*\\.jar$|.*javax.servlet.jsp.jstl-[^/]*\\.jar|.*taglibs-standard-impl-.*\\.jar");
```

Comment:
what's the motivation for changing the pattern for the servlet-api? Should such a change be made to all other patterns too?

Reply:
reverted.